gateleen-scheduler/src/main/java/org/swisspush/gateleen/scheduler/SchedulerResourceManager.java

```diff
@@ -177,4 +183,51 @@ public void refresh() {
     public void enableResourceLogging(boolean resourceLoggingEnabled) {
         this.logConfigurationResourceChanges = resourceLoggingEnabled;
     }
+
+
+    /**
+     * Evaluate if the given date is in daylight saving time or not.
+     *
+     * @return true if in daylight saving time, false otherwise
+     */
+    public static boolean isDaylightSavingTimeState(ZonedDateTime checkDate) {
+        return checkDate.getZone().getRules().isDaylightSavings(checkDate.toInstant());
+    }
+
+    /**
+     * Evaluate if the current date is in daylight saving time or not.
+     *
+     * @return true if in daylight saving time, false otherwise
+     */
+    public static boolean isCurrentDaylightSavingTimeState() {
+        return isDaylightSavingTimeState(ZonedDateTime.now());
```

Comment:
See [here](https://github.com/swisspost/gateleen/pull/694/files#r2412744651)

Reply:
It is by purpose like this. We do not stick with any Timezone, it should work wherever the system runs and behaves based on the local timezone.